packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -80,23 +90,48 @@ class RenderableTiledMap {
         result[src] = await SpriteBatch.load(src);
       }
     });
+
+    return result;
+  }
+
+  static Future<Map<String, SpriteBatch>> _deepCopySpriteBatchMap(
+    Map<String, SpriteBatch> copyFrom,
+  ) async {
+    final result = <String, SpriteBatch>{};
+
+    await Future.forEach(copyFrom.keys, (String src) async {
+      result[src] = await SpriteBatch.load(src);
+    });
```

Comment:
This could probably be written a bit simpler I think?
Something like:
```dart
    return {
      for (var key in copyFrom.keys) key: await SpriteBatch.load(key)
    };
```

Reply:
Great point, just updated!